src/main/java/org/dehydrogenaza/data/DiseaseEnum.java

```diff
@@ -0,0 +1,16 @@
+package org.dehydrogenaza.data;
+
+
+public enum DiseaseEnum {
```

Comment:
Wg konwencji nie daje się słówka "enum" w nazwie enumów.
```suggestion
public enum Disease {
```

Reply:
ok.